taxcalc/functions.py

```diff
@@ -1459,77 +1488,47 @@ def AddCTC(_nctcr, _precrd, c07220, e00200, e82882, e30100, _sey, _setax,
     # Part I of 2005 form 8812
     if _nctcr > 0:
         c82925 = _precrd
-
         c82930 = c07220
-
         c82935 = c82925 - c82930
-
         # CTC not applied to tax
-
-        c82880 = max(0., e00200 + e82882 + e30100 + max(0., _sey) -
-                     0.5 * _setax)
+        c82880 = max(0, _earned)
         if _exact == 1:
             c82880 = e82880
-
-        h82880 = c82880
-
+        h82880 = 0
         c82885 = max(0., c82880 - ACTC_Income_thd)
-
         c82890 = ACTC_rt * c82885
-
-    else:
-        c82925, c82930, c82935, c82880, h82880, c82885, c82890 = (0., 0., 0.,
-                                                                  0., 0., 0.,
-                                                                  0.)
+    # else:
+    #    c82925, c82930, c82935, c82880, h82880, c82885, c82890 = (0., 0., 0.,
+    #                                                              0., 0., 0.,
+    #                                                              0.)
 
     # Part II of 2005 form 8812
 
     if _nctcr >= ACTC_ChildNum and c82890 < c82935:
         c82900 = 0.0765 * min(SS_Earnings_c, c82880)
-
         c82905 = float((1 - ALD_SelfEmploymentTax_HC) * e03260 + e09800)
-
         c82910 = c82900 + c82905
-
         c82915 = c59660 + e11200
-
         c82920 = max(0., c82910 - c82915)
         c82937 = max(c82890, c82920)
-
-    else:
-
-        c82900, c82905, c82910, c82915, c82920, c82937 = 0., 0., 0., 0., 0., 0.
+    # else:
+    #   c82900, c82905, c82910, c82915, c82920, c82937 = 0., 0., 0., 0., 0., 0.
 
     # Part II of 2005 form 8812
-    if _nctcr > 2 and c82890 >= c82935:
-        c82940 = c82935
-
-    if _nctcr > 2 and c82890 < c82935:
-        c82940 = min(c82935, c82937)
-
-    if _nctcr <= 2 and c82890 > 0:
+    if _nctcr > 0 and _nctcr <= 2 and c82890 > 0:
```

Comment:
`_nctcr  =  n24` is the "Number of Children for Child Tax Credit". If `_nctcr = 0`, then `c82940`, additional Child Tax Credit Amt, shall not be concerned.